mapfishapp/src/main/webapp/app/js/GEOR_mapinit.js

```diff
@@ -340,6 +350,67 @@ GEOR.mapinit = (function() {
                 }
             }
         });
+        
+        // support for JSON search parameter
+        if( initSearch.hasOwnProperty('typename') && initSearch.hasOwnProperty('featurens') 
+            && initSearch.hasOwnProperty('owsurl') && initSearch.hasOwnProperty('cql_filter') ) {
+            var record = {
+                typeName: initSearch.typename,
+                featureNS: initSearch.featurens,
+                owsURL: initSearch.owsurl,
+                WFSversion : "1.1.0"
```

Comment:
Reading again the code behind GEOR.ows.WFSDescribeFeatureType, you should not have to provide a version number, it we be automatically negociated, since extractFeatureNS is true.

You also do not need featureNS if you provide a fully qualified typename (ie with nsalias included)